content/post/analytics-engineer.md

```diff
@@ -0,0 +1,52 @@
+---
+author: Michael Kaminsky
+title: "The Analytics Engineer"
+publishDate: "2019-01-31"
+draft: True
+tags: 
+ - analytics
+ - software
+ - data
+ - engineering
+---
+
+The landscape of the data and analytics world is shifting rapidly. In many companies, the roles and responsibilities  of data engineers, analysts, and data scientists are changing. This change has created the need for a new role on the data team which some of have taken to calling the "analytics engineer". 
+
+<!--more-->
+
+The analytics engineer sits at the intersection of the skill sets of data scientists, analysts, and data engineers. They bring a formal and rigorous software engineering practice to the efforts of analysts and data scientists, and they bring an analytical and business-outcomes mindset to the efforts of data engineering. It's their job to build tools and infrastructure to support the efforts of the analytics and data team as a whole.
+
+Before we dive further into the role, we should cover some background on the "traditional" roles on the data team[^1]. 
+
+* Data engineers: traditionally, this has been a "plumbing" job of moving bytes from point A to point B. They were concerned with building robust and scalable infrastructure for ingesting and storing data, but generally did not concern themselves with "business logic" -- once the data were in the warehouse, it wasn't there problem any more.
```

Comment:
Most software engineers that I talk to think ETL is the primary task (or short hand) that Data Engs do. May be worth giving that a nod (as unappealing as that might seem).

Reply:
Interesting! I've had the opposite experience 😬 What do you think the appropriate edit here would be that reflects this?

Reply:
🤷‍♂️ 

* Data engineers: traditionally, this has been a "plumbing" job of moving bytes from point A to point B, typically misnomered simply as "ETL". They were....

Reply:
Ah, got it! This was not how I had interpreted your initial comment but this makes sense to me. Adding in!